integrade/tests/api/v1/test_user_accounts.py

```diff
@@ -129,3 +129,22 @@ def test_change_password():
 
     response = client.post(urls.AUTH_TOKEN_DESTROY, {}, auth=auth)
     assert response.status_code == 204
+
+
+def test_cannot_delete_self():
```

Comment:
Since this is a documented bug maybe we should mark it with xfail (`@pytest.mark.xfail`). That way the travis build will pass but we can have it on the books that this unfortunate behavior is present.
https://docs.pytest.org/en/reorganize-docs/new-docs/user/xfail.html